module/documents/activity/mixin.mjs

```diff
@@ -31,6 +31,32 @@ export default Base => class extends PseudoDocumentMixin(Base) {
     }
   });
 
+  /* -------------------------------------------- */
+
+  /**
+   * Perform the pre-localization of this data model.
+   */
+  static localize() {
+    Localization.localizeDataModel(this);
+    const fields = this.schema.fields;
+    if ( fields.damage?.fields.parts ) this._localizeSchema(fields.damage.fields.parts.element, ["DND5E.DAMAGE"]);
```

Comment:
Should this be a part of an `AttackActivity.localize` override?

Reply:
I have it here because other activities like `SaveActivity` will also have damage in the same place, so it seemed simpler to have it here rather than have each activity with damage parts localize it on their own.